### PR TITLE
[4] Smart Search: Search Filters Empty State

### PR DESCRIPTION
**Pull request by PhilETaylor**
Pull Request for Issue https://github.com/joomla/joomla-cms/issues/33670

### Summary of Changes

Empty State page for Smart Search: Search Filters

(yours may be blue, mine is custom colours)

Screenshot 2021-05-10 at 12 26 47

### CODE REVIEW

**Comment by PhilETaylor** [comment]
original colour scheme restored

Screenshot 2021-05-10 at 12 31 39

File: administrator/language/en-GB/com_finder.ini [hunk 1]
```diff
@@ -85,6 +85,9 @@ COM_FINDER_FILTER_WHEN_END_DATE_LABEL="When (End Date)"
 COM_FINDER_FILTER_WHEN_EXACTLY="Exactly"
 COM_FINDER_FILTER_WHEN_START_DATE_LABEL="When (Start Date)"
 COM_FINDER_FILTERS="Filters"
+COM_FINDER_FILTERS_EMPTYSTATE_BUTTON_ADD="Add a filter"
+COM_FINDER_FILTERS_EMPTYSTATE_CONTENT="You have not yet created any filters."
+COM_FINDER_FILTERS_N_ITEMS_CHECKED_IN="%d filters checked in."
```
Comment (Kostelano) [inline administrator/language/en-GB/com_finder.ini:90]: Duplicate language string

Comment (PhilETaylor) [reply administrator/language/en-GB/com_finder.ini:90]: New keyboard and fat fingers. Thanks.

**Comment by Kostelano** [comment]
I have tested this item :white_check_mark: successfully on c555ced57a7acff20f0a5cc420be0316b22146a6<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/33733">issues.joomla.org/tracker/joomla-cms/33733</a>.</sub>

**Comment by ghost** [comment]
I have tested this item :white_check_mark: successfully on c555ced57a7acff20f0a5cc420be0316b22146a6<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/33733">issues.joomla.org/tracker/joomla-cms/33733</a>.</sub>

**Comment by richard67** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/33733">issues.joomla.org/tracker/joomla-cms/33733</a>.</sub>

**Comment by Quy** [comment]
Thank you!

**Pull request merged**